test-case/run-all-tests.sh

```diff
@@ -21,8 +21,6 @@ very_short_tests()
 
 
 testlist="
-firmware-presence
-firmware-load
```

Comment:
sounds good for now, we are missing some parts in the kernel/firmware interaction. It's ok to remove this firmware-load for now and re-add it later when the IPC4 support actually enables it.